docker-compose.dev.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
```

Comment:
probably at this point we should remove the existing `docker-compose.override.yml` in favor of this